### DISCUSSION TITLE
Add information about reporters of closed issues

### DISCUSSION THREAD

**Discussion by srenatus** [community/community#6033] [Other Feature Feedback, Questions, & Ideas] [answered]
We want to show that we value users contributing by opening issues and reporting bugs.

It would be nice if the auto-generated release notes could account for that, like

> * Fix unexpected floob when flobbing by @committer in [PR], reported by @reporter

**Comment by MylesBorins** [comment, marked as answer]
This is definitely a cool idea but not something that we are going to be able to support at this time. In the future we may want to explore syntax to allow folks to customize the titles of each individual entry in the release notes and expanding too much functionality today could make offering that support significantly more complicated.

If we do an additional set of work on configuration in the future we can revisit this suggestion, thank you for bringing it up!
